style.css

```
@@ -0,0 +1,273 @@
+:root {
+    --primary-main: #0067fc;
+    --primary-light: #f0f8ff;
+    --primary-dark: #0048cd;
+    --white-main: #fff;
+
+    --font-size-extra-small: 0.75rem;
+    --font-size-small: 0.875rem; 
+    --font-size-medium: 1rem; 
+    --font-size-large: 1.5rem;
+
+    --size-1: 0.25rem;
+    --size-2: 0.5rem; 
+    --size-3: 1rem; 
+    --size-4: 2rem; 
+
+    --transition-duration: 0.3s;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+}
+
+.container {
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+header {
+    position: sticky;
+    height: 65px;
+    top: 0;
+    left: 0;
+    background-color: var(--white-main);
+    padding-right: var(--size-2);
+}
+
+.header-container {
+    display: flex;
+    justify-content: space-between;
+}
+
+header .logo {
+    width: 6.25rem;
+}
+
+.button {
+    cursor: pointer;
+    border: none;
+    background: none;
+    height: 2.1rem;
+    font-size: var(--font-size-extra-small);
+    color: var(--white-main);
+    background-color: var(--primary-main);
+    border-radius: var(--size-1);
+    transition: color var(--transition-duration);
+}
+
+.button:hover {
+    color: var(--primary-main);
+    background-color: var(--white-main);
+    border: 1px solid var(--primary-main);
+}
+
+nav {
```

Same situation here with element-based styling. Recommendation:

```suggestion
.nav {
```